app/src/main/java/nextstep/payments/model/CardUiState.kt

```diff
@@ -1,199 +1,7 @@
-                                color = Color.White,
-                                fontSize = 12.sp
-                            )
-                            Text(
-                                text = card.expiredDate,
-                                color = Color.White,
-                                fontSize = 12.sp
-                            )
-                        }
-                    }
-                }
-            )
-        }
-        item {
-            content()
-        }
-    }
-}
-
-@Preview(showBackground = true)
-@Composable
-fun CardListPreview() {
-    CardList(
-        cards = listOf(
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFFF04651,
-                cardCompany = "BC카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFF0E19ED,
-                cardCompany = "신한카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFFF0DE1F,
-                cardCompany = "카카오뱅크"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFF030201,
-                cardCompany = "현대카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFF416CE0,
-                cardCompany = "우리카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFFED2405,
-                cardCompany = "롯데카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFF0CAB73,
-                cardCompany = "하나카드"
-            ),
-            Card(
-                cardNumber = "1234-0000-1111-1234",
-                expiredDate = "12/34",
-                ownerName = "홍길동",
-                password = "1234",
-                color = 0xFF695F54,
-                cardCompany = "국민카드"
-            ),
-        )
-    )
+    data object Empty : CardUiState
+    data object One : CardUiState
+    data object Many : CardUiState
```

Comment:
아래 처럼 구성한다면, 각 UiState 타입에 맞춰서 원하는 데이터를 들고 있게끔 구성할 수 있겠어요 :)
일일이 List\<cards>가 필요한 곳에 emptyList등을 억지로 채우거나, 1개인데 List로 표현해야할 일이 사라질 거라 기대돼요 :)
```suggestion
    data object Empty : CardUiState
    data class One(val card: Card) : CardUiState
    data class Many(val cards: List<Card>) : CardUiState
```